hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/RemoteException.java

```diff
@@ -47,12 +50,23 @@ public RemoteException(String className, String msg) {
    * @param erCode may be null
    */
   public RemoteException(String className, String msg, RpcErrorCodeProto erCode) {
+    this(className, msg, erCode, null);
+  }
+
+  /**
+   * @param className wrapped exception, may be null
+   * @param msg may be null
+   * @param erCode may be null
+   */
+  public RemoteException(String className, String msg, RpcErrorCodeProto erCode,
+                         ExceptionReconstructParamsProto paramsProto) {
     super(msg);
     this.className = className;
     if (erCode != null)
```

Comment:
good time to add {} here, or move to ? : instead. I know its old code, but it is adjacent to your work

Reply:
Updated.